python/tests/core/constraints/factories/test_count_metrics.py

```diff
@@ -1,10 +1,21 @@
 from whylogs.core.constraints.factories import (
     count_below_number,
+    no_missing_values,
     null_percentage_below_number,
     null_values_below_number,
 )
 
 
+def test_no_missing_values(builder):
+    builder.add_constraint(no_missing_values(column_name="animal"))
+    builder.add_constraint(no_missing_values(column_name="weight"))
+    constraint = builder.build()
+    assert constraint.validate() is False
```

Comment:
if it passes on having no_missing_values, why is `constraints.validate()` False?

Reply:
constraints.validate() is False if any constraint in `constraints` fails. Since `weight` fails on the check, validate() returns false.
actually, `constraint` should be in plural here.